packages/datatrak-web/src/components/Button.tsx

```diff
@@ -16,9 +18,10 @@ const StyledButton = styled(UIButton)`
   }
 `;
 
-const TooltipButtonWrapper = styled.span`
+export const TooltipButtonWrapper = styled.span`
```

Comment:
Exporting so this wrapper can be explicitly selected by other components that use `Button`, rather than using `& > span` selector, which could affect sibling components